src/router/index.ts

```diff
@@ -18,7 +19,20 @@ const routes: Array<RouteConfig> = [
     // which is lazy-loaded when the route is visited.
     component: () =>
       import(/* webpackChunkName: "about" */ "../views/About.vue")
-  }
+  },
+  {
+    path: "/profile",
+    name: "Profile",
+    component: () =>
+      import(/* webpackChunkName: "Profile" */ "../views/CreatePost.vue")
```

Comment:
seems like you switch these components. This should ..views/Profile.vue and vc for the Post route underneath